GoogleDataTransportCCTSupport/GDTCCTTests/Unit/Helpers/GDTCCTEventGenerator.m

```diff
@@ -55,6 +54,15 @@ - (void)deleteGeneratedFilesFromDisk {
   }
 }
 
+- (void)writeEvent:(GDTCOREvent *)event toGDTPath:(NSString *)path error:(NSError **)error {
+  SEL sel = NSSelectorFromString(@"writeToGDTPath:error:");
+  IMP imp = [event methodForSelector:sel];
+  if (imp) {
```

Comment:
I wonder if using [respondsToSelector:](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418583-respondstoselector?language=objc) will work for this case? It would look a bit more clean and Objective C friendly. 

Another question, why the event may not provide an implementation for the method?

Reply:
It would work for the check, but I have to get the IMP anyway, so might as well check that (in test code).

Reply:
In this particular method you can go without IMP e.g.:
```
[(id)event writeToGDTPath:path error:error];
```